docs/usage/models.md

```diff
@@ -319,6 +319,14 @@ To inherit from a GenericModel without replacing the `TypeVar` instance, a class
 ```
 _(This script is complete, it should run "as is")_
 
+You can also create a generic subclass of a `GenericModel` that partially or fully replaces the type 
```

Comment:
The example only goes into the "fully replaced" version.  Does a partial replacement also work or is that dependent on #1989  and/or https://github.com/samuelcolvin/pydantic/issues/2002?

If it really does only work inheriting from a fully concrete class, I'd probably leave out this doc section (as that would be the expected behavior anyway I would think).

Reply:
I'm guessing the partially replaced version depends on #1989

Reply:
I can remove it if you think it's superfluous.